cdisc_rules_engine/services/reporting/base_report.py

```diff
@@ -37,9 +37,8 @@ def get_summary_data(self) -> List[List]:
         information:
         return [
             "Dataset",
-            "RuleID",
+            "CORE-ID",
             "Message",
-            "Executability",
```

Comment:
Are we supposed to be removing executability? I don't remember seeing it in the acceptance criteria

Reply:
Question for @nickdedonder
Your provided template has `Executability` removed from the issue summary. It still exists in the Issue details. Is this intentional?

Reply:
@nickdedonder 
Sorry, to clarify, this was mislabeled as `Severity` in the previous version of the report template. So you actually removed the `Severity` column.

Reply:
yes, I deleted that as there was a decision not to use severity. We'll check in the future what do with it as this is only applicable for the PMDA. I don't want to have severity always present if I'm not running rules for the PMDA